Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -378,9 +378,9 @@ def audit_license
           end
         end
 
-        return unless @online
+        return unless @online && !formula.versioned_formula?
```

Comment:
Why `!formula.versioned_formula?`? Also, please refactor to not use `unless .. &&` (or `unless .. ||`)

Reply:
> This also excludes checking versioned formulae because the GitHub API only returns the license for the default branch and the license information for a versioned branch may be different (see `scw@1` vs `scw`).

Reply:
Cool. In that case: I think this warrants a comment.